Gemfile.lock

```diff
@@ -600,7 +600,7 @@ GEM
       link_header (~> 0.0, >= 0.0.8)
     rdf-normalize (0.7.0)
       rdf (~> 3.3)
-    rdoc (6.6.2)
+    rdoc (6.6.3.1)
```

Comment:
This is a security update. https://www.ruby-lang.org/en/news/2024/03/21/rce-rdoc-cve-2024-27281/